csrc/scheduler/matmul.cpp

```diff
@@ -437,6 +463,102 @@ void scheduleProlog(TensorView* shared_mem_tv, const MatmulParams& params) {
           .propagateParallelType());
 }
 
+void schedule_output_tensor(
```

Comment:
According our coding style, it should be `scheduleOutputTensor`

Reply:
Are you going to update this?

Reply:
updated.